src/Microsoft.Health.Fhir.Anonymizer.Shared.AzureDataFactoryPipeline/src/DataFactoryCustomActivity.cs

```diff
@@ -220,10 +220,10 @@ private async Task AnonymizeSingleBlobInNdJsonFormatAsync(BlobClient inputBlobCl
             progress.ProgressChanged += (obj, args) =>
             {
                 Interlocked.Add(ref processedCount, args.ProcessCompleted);
-                Interlocked.Add(ref processedErrorCount, args.ProcessFailed);
+                Interlocked.Add(ref skippedCount, args.ProcessSkipped);
                 Interlocked.Add(ref consumedCount, args.ConsumeCompleted);
```

Comment:
need a doc to explain these three numbers

Reply:
We can refine these three numbers later.